crates/dprint/src/main.rs

```diff
@@ -40,6 +40,12 @@ fn main() {
 
 async fn run(runtime_handle: tokio::runtime::Handle) -> Result<(), AppError> {
   let args = arg_parser::parse_args(std::env::args().collect(), RealStdInReader)?;
+
+  #[cfg(feature = "completions_generator")]
+  if let arg_parser::SubCommand::GenerateCompletions = args.sub_command {
+    std::process::exit(0);
```

Comment:
```suggestion
    return Ok(());
```

Can we just do this?